chore(learn): add details about grading tasks (#29139)
mdn/content: files/en-us/mdn/community/index.md

## Ways to contribute

Here's a list of ways you can contribute to MDN Web Docs:

- [Fixing known high impact issues](https://github.com/orgs/mdn/projects/25/views/1)
- [Reviewing pull requests](/en-US/docs/MDN/Community/Pull_requests)
- [Help beginners to learn on MDN Web Docs](/en-US/docs/MDN/Community/Learn_forum)
- [Contribute to MDN Web Docs interactive examples](https://github.com/mdn/interactive-examples/blob/main/CONTRIBUTING.md)
- [Help translate MDN Web Docs](/en-US/docs/MDN/Community/Contributing/Translated_content)
- [Help fix known platform issues](https://github.com/mdn/yari/issues)
- [Help us keep browser compatibility data up to date](https://github.com/mdn/browser-compat-data)

## Community resources

- [Communication channels](/en-US/docs/MDN/Community/Communication_channels)

  - : This page lists communication channels used by the MDN team and our community, with hints on which might be best for you.

- [Contributing](/en-US/docs/MDN/Community/Contributing)

  - : This section explains how you can start contributing and the type of contributions we accept.

>     This section covers [Getting started](/en-US/docs/MDN/Community/Contributing/Getting_started), [Our repositories](/en-US/docs/MDN/Community/Contributing/Our_repositories), [Translated content](/en-US/docs/MDN/Community/Contributing/Translated_content), and [Security vulnerability response steps](/en-US/docs/MDN/Community/Contributing/Security_vulnerability_response).

Reviewer: ```suggestion
    It covers [Getting started](/en-US/docs/MDN/Community/Contributing/Getting_started), [Our repositories](/en-US/docs/MDN/Community/Contributing/Our_repositories), [Translated content](/en-US/docs/MDN/Community/Contributing/Translated_content), and [Security vulnerability response steps](/en-US/docs/MDN/Community/Contributing/Security_vulnerability_response).
```
